cmd/ch-k8s-lbaas-controller/ch-k8s-lbaas-controller.go

```diff
@@ -90,13 +91,17 @@ func main() {
 	servicesInformer := kubeInformerFactory.Core().V1().Services()
 	nodesInformer := kubeInformerFactory.Core().V1().Nodes()
 	endpointsInformer := kubeInformerFactory.Core().V1().Endpoints()
+	networkPoliciesInformer := kubeInformerFactory.Networking().V1().NetworkPolicies()
+	podsInformer := kubeInformerFactory.Core().V1().Pods() // TODO: I don't want to be informed about pods. Just need to list them
```

Comment:
If I recall how the kubernetes golang stuff works correctly, you need an informer to list stuff properly.

Reply:
Alright, yes, as far as I understand it now, this seems to be the case.